ui/cypress/e2e/tasks.test.ts

```diff
@@ -41,12 +41,13 @@ from(bucket: "${name}"{rightarrow}
       .should('have.length', 1)
       .and('contain', taskName)
 
-    // TODO: extend to create from template overlay
-    cy.getByTestID('add-resource-dropdown--button').click()
-    cy.getByTestID('add-resource-dropdown--template').click()
-    cy.getByTestID('task-import-template--overlay').within(() => {
-      cy.get('.cf-overlay--dismiss').click()
-    })
+    // Creating a task from a template is no longer an option - 
+    // see https://github.com/influxdata/ui/issues/869
+    // cy.getByTestID('add-resource-dropdown--button').click()
+    // cy.getByTestID('add-resource-dropdown--template').click()
+    // cy.getByTestID('task-import-template--overlay').within(() => {
+    //   cy.get('.cf-overlay--dismiss').click()
+    // })
```

Comment:
you can go ahead and delete this bit unless there's a reason to keep it around

Reply:
Will do, thanks!